starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,72 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>Apple Pie Recipe</title>
-</head>
-<body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <head>
+    <title>Apple Pie Recipe</title>
+    <link
+      rel="stylesheet"
+      href="/starter-code/apple-pie-starter-code/css/main.css"
+    />
+  </head>
+  <body class="body-align">
+    <div class="image-manipulation-upper">
+      <h1 id="title-color" class="centered">Apple Pie</h1>
+    </div>
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+    <div class="page-margin">
+      <div>
+        <p>
+          This was my grandmother's apple pie recipe. I have never seen another
+          one quite like it. It will always be my favorite and has won me
+          several first place prizes in local competitions. I hope it becomes
+          one of your favorites as well!
+        </p>
+        <div class="image-manipulation-lower">
+          <img
```

Comment:
images should have a certain size, so maybe try to play around with `width` in css